tests/test_userspace.py

```diff
@@ -131,10 +147,9 @@ class Clk:
         ("ethptp_k", 125000000, ("5800a000.ethernet",)),
         ("ethck_k", 125000000, ("5800a000.ethernet",)),
         ("ethrx", 125000000, ("5800a000.ethernet",)),
-
         # CAN Clock: Chosen to be 48MHz for minimum baudrate error across all rates
         ("fdcan_k", 48000000, ("4400f000.can", "4400e000.can")),
-    )
+    ),
```

Comment:
Some changes here got into the wrong commits. Please rebase those into the first commit.